drafting.typ

```diff
@@ -640,7 +640,7 @@
 
     // Overwrite the properties for left / right margins
     // This way we only need to calculate this once
-    if page.margin != auto and "inside" in page.margin.keys() {
+    if type(page.margin) != relative and page.margin != auto and "inside" in page.margin.keys() {
```

Comment:
I think it would be easier to make a call to `_get-margins()` for normalizing here